# Sales Data Analysis (Mini Project)

Goal: Analyze sales data to find top cities, categories, products, and overall profitability.


In [9]:
from pathlib import Path
import pandas as pd

BASE_DIR = Path(r"C:\Users\Dunhi\Desktop\python\Data Analysis")
df = pd.read_csv(BASE_DIR / "sales.csv")


In [10]:
city_summary = (
    df.groupby("City")[["Sales", "Profit"]]
      .sum()
      .sort_values("Sales", ascending=False)
)

city_summary["Profit_Margin_%"] = (city_summary["Profit"] / city_summary["Sales"]) * 100
city_summary


Sales  Profit  Profit_Margin_%
City                                  
Dammam   5660     680        12.014134
Riyadh   5100     595        11.666667
Jeddah   1150     155        13.478261

In [11]:
city_summary.sort_values("Profit_Margin_%", ascending=False)


Sales  Profit  Profit_Margin_%
City                                  
Jeddah   1150     155        13.478261
Dammam   5660     680        12.014134
Riyadh   5100     595        11.666667

## Key Insights
- Dammam has the highest total sales and profit.
- Technology is the top category in both sales and profit.
- Laptop is the best-selling and most profitable product.
- Total Sales = 11,910 and Total Profit = 1,430 (~12% profit margin).
